plugins/livelog/livelog.go

```diff
@@ -100,11 +104,46 @@ func (tp *taskPlugin) Finished(success bool) error {
 		return err
 	}
 	defer file.Close()
+
+	tempFile, err := ioutil.TempFile(os.TempDir(), "livelog")
+
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		if err2 := tempFile.Close(); err2 != nil {
+			tp.context.LogError(err2)
```

Comment:
Don't log this error to the task log!

1) You can't log to task log in Finished() - the log is closed!
2) This is clearly not the tasks fault, it's an internal worker error, the task should not ever get system error messages from the worker. These should go into logrus, for syslog...